services/terraformer/templates/oci.tpl

```diff
@@ -161,14 +161,21 @@ resource "oci_core_instance" "{{ $node.Name }}" {
         - iptables -Z
         # Make changes persistent
         - netfilter-persistent save
+        {{- if not $nodepool.IsControl }}
+        # Mount volume
+        - mkdir -p /opt/claudie/data
+        - mkfs.xfs /dev/sdb
```

Comment:
can we be sure that the "storage" disk will always be recognised as `sdb`?
would there be an option to get a UUID from GCP or `/dev/disk/by-id` or sth?